raiden/constants.py

```diff
@@ -32,6 +32,12 @@ class EthClient(Enum):
 
 DISCOVERY_TX_GAS_LIMIT = 76000
 
+# The more pending transfers there are, the more computationally complex
+# it becomes to unlock them. The maximal number defined here cannot be
+# exceeded lest we run out of gas trying to unlock.
```

Comment:
Please add to the comment that this value is inclusive

Reply:
Done,  I added the gas limit as a constant too.